Pharmacy/Pharmacy/Services/HospitalAuthService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Pharmacy.Model;
+using Pharmacy.Repositories;
+using Pharmacy.Repositories.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pharmacy.Services
+{
```

Comment:
This doesn't have to be a service since its whole logic is based on the model. Probably better to have it in the IHospitalRepository where we could validate the ApiKey